tests/unit/keyspace.tcl

```diff
@@ -13,6 +13,13 @@ start_server {tags {"keyspace"}} {
         list [r del foo1{t} foo2{t} foo3{t} foo4{t}] [r mget foo1{t} foo2{t} foo3{t}]
     } {3 {{} {} {}}}
 
+    test {Untagged multi-key commands} {
+        r set foo1 a
+        r set foo2 b
+        r set foo3 c
+        list [r del foo1 foo2 foo3 foo4] [r mget foo1 foo2 foo3]
```

Comment:
that's a poor test of MGET, maybe change the order MGET before DEL?
maybe also use MSET?